js/src/lib/interfaces/Types.ts

```diff
@@ -466,9 +466,14 @@ export const PIPELINE_DATA = ensureRecordOfPipelines({
 		],
 		color:        "blue",
 	},
+	"tabular-regression": {
```

Comment:
@lhoestq do we want to hide this in datasets or is it good?

Reply:
I think it's fine, thanks !